runtime/lua/vim/shared.lua

```diff
@@ -12,8 +12,8 @@ local vim = vim or {}
 --- same functions as those in the input table. Userdata and threads are not
 --- copied and will throw an error.
 ---
----@param orig Table to copy
----@returns New table of copied keys and (nested) values.
+---@param orig table Table to copy
+---@return table #New table of copied keys and (nested) values.
```

Comment:
what's with the pound sign? If this is in the emmy lua spec, we don't traditionally use it

Reply:
Comment. At least sumneko's lsp has this effect on the hover display.

https://github.com/sumneko/lua-language-server/wiki/EmmyLua-Annotations#comments

It maybe only @ in the original.

https://emmylua.github.io/annotations/return.html?highlight=comment

Reply:
Traditionally, we use the wrong grammars, such as `@returns` and `@param ...`.